bool kinrequest
---
float64[3] p
float64[4] q
float64[3] p2
float64[4] q2
float64[6] J1
float64[6] J2
float64[6] J3
float64[6] J4
float64[6] J5
float64[6] J6

